co2calculator/api/energy.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Energy classes"""
+from co2calculator import (
+    CountryCode2,
+    CountryCode3,
+    calc_co2_electricity,
+    ElectricityFuel,
+    calc_co2_heating,
+)
+
+from typing import Optional
+
+
+class Energy:
+    def __init__(
+        self,
+        consumption: float,
+        fuel_type: Optional[str] = None,
+        unit: Optional[str] = None,
+        own_share: float = 1.0,
+        country_code: Optional[str] = None,
+    ):
+        """Initialize an Energy object"""
```

Comment:
add parameter descriptions

Reply:
thanks, added, please also check if you agree moving `country_code` and `unit` to the class methods

Reply:
yes I think it's better like this too. Thanks!